client/src/app/core/repositories/users/user-repository.service.ts

```diff
@@ -234,12 +247,23 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User, UserTi
         return await this.httpService.post<MassImportResult>(`/rest/users/user/mass_import/`, { users: data });
     }
 
+    public async update(update: Partial<User>, viewModel: ViewUser): Promise<void> {
+        this.preventAlterationOnAdmin(viewModel);
+        return super.update(update, viewModel);
+    }
+
+    public async delete(viewModel: ViewUser): Promise<void> {
+        this.preventInDemo();
```

Comment:
Why can you create users  but not delete them?

Reply:
even tho you would accidentally create users, they DB resets every night, so it does not really matter. Randomly deleting users is not really required. Deleting manually created users, on the other hand (simple vandalism) could have annoying effects while playing around (with i.e voting).

Reply:
ok